specs/order_spec.rb

```diff
@@ -78,33 +80,44 @@
   end
 end
 
-# TODO: change 'xdescribe' to 'describe' to run these tests
-xdescribe "Order Wave 2" do
+describe "Order Wave 2" do
   describe "Order.all" do
     it "Returns an array of all orders" do
-      # TODO: Your test code here!
+      result = Grocery::Order.all
+      result.must_equal (CSV.read("/support/orders.csv"))
     end
 
     it "Returns accurate information about the first order" do
-      # TODO: Your test code here!
+      result = Grocery::Order.all
+      result_first = result[0]
+      test_result = CSV.read("/support/orders.csv")
+      test_first = test_result[0]
+      result_first.must_equal test_first
```

Comment:
Again you're not creating an Order instance , or testing to see if the `id` field matches the ID from the CSV.